.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Great job adding `node_modules` to `.gitignore` file, please add `test.md` as well and your file will meet all the requirements